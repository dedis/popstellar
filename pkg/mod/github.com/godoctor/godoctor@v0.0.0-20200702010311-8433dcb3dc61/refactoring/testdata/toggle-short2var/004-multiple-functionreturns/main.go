// <<<<< toggle,11,3,11,13,pass
package main

import "fmt"

func f() (int, float64) {
	return 1, 2.3
}

func main() {
  i, x := f()
  fmt.Println(i, x)
}
