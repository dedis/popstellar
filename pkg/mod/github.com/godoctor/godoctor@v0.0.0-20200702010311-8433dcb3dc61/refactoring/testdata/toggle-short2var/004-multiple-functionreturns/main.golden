// <<<<< toggle,11,3,11,13,pass
package main

import "fmt"

func f() (int, float64) {
	return 1, 2.3
}

func main() {
  var i, x = f()
  fmt.Println(i, x)
}
